docs/xxprintcodecache.md

```diff
@@ -0,0 +1,46 @@
+<!--
+* Copyright (c) 2020, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]PrintCodeCache
+
+This Oracle HotSpot option prints the code cache memory usage when the application exits.
+
+## Syntax
+
+        -XX:[+|-]PrintCodeCache
+
+| Setting                      | Effect  | Default                                                                            |
+|------------------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+PrintCodeCache`        | Enable  |                                                                                    |
+| `-XX:-PrintCodeCache`        | Disable | <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span>     |
+
+As discussed in the Oracle documentation, the code cache usage can be shown when the application exits, by specifying `–XX:+PrintCodeCache` on the java launcher command line. The output looks similar to the following: 
+
+```
+CodeCache: size=262144Kb used=454Kb max_used=457Kb free=261690Kb
+```
+
+For more information about the output, see [Codecache Tuning](https://docs.oracle.com/javase/8/embedded/develop-apps-platforms/codecache.htm).
```

Comment:
Hmm, our output is not as comprehesive as theirs, so should we send users to Oracle documentation in this case?

Reply:
Will remove the link and add an explanation to each item.